tests/test_license.py

```diff
@@ -0,0 +1,25 @@
+"""
+Tests of the ``license`` field in ``pyproject.toml``, which can be populated
+from setuptools' ``license`` field.
+"""
+
+# setuptools *ONLY* supports bare license strings in the `license` field.  There exists a
+# `license_files` and a deprecated `license_file` field.  The latter simply raises a warning and the
+# content is prepended to `license_files`.  Neither populates the `license` property.
+# `license_files` is the subject of a PEP: https://peps.python.org/pep-0639/
```

Comment:
I think it would be useful to add a test that sets the `license_files` field in this file, and probably one for `license_file` as well, because IMO it's a good idea to document and constraint the expected behavior of the plugin when one of those fields is present, even if all it shows is that they don't (yet) show up in `pyproject.toml`.

Reply:
I didn't see a lot of point in a test that shows `license_files` (and by extension, `license_file`) are consigned to `/dev/null`.  I'm thinking this probably belongs in a separate `test_license_files.py`.

Reply:
I've added a couple for `license_files`; one for literal file names, the other for file path globs.

I've omitted `license_file`, since it's deprecated in `setuptools` now, _and_ it copies the data into `license_files` for us, so if we support the latter, implicitly the former will work too.

Both these tests are marked `xfail` as we do not yet support this feature.